packages/vuetify/src/components/VAlert/VAlert.ts

```diff
@@ -148,10 +148,10 @@ export default mixins(
       if (typeof this.icon === 'string' && this.icon) return this.icon
 
       switch (this.type) {
```

Comment:
This `switch` and more globally the `computedIcon` function can probably can be simplified/shortened.
Maybe :
```ts
computedIcon (): string | boolean {
  return this.icon || (['info', 'error', 'sucess', 'warning'].includes(this.type) && `$${this.type}`)
}
```

The thing I don't know is : do users use `<v-alert type="info" icon="false">Message</v-alert>` behavior ?